src/image_bios.cpp

```diff
@@ -5,12 +5,19 @@
 
 #include "config.h"
 
+#include "image_bios.hpp"
```

Comment:
Dow we have different clang-format? My version moved this line in 14a99fc798411580f4df55b77a451602940c6266.

Reply:
Probably yes. I use:
```
clang-format version 10.0.1 (Fedora 10.0.1-3.fc32)
```
And I agree with it, the header with the same name should be always on top.

Reply:
Yeah, but version 11 does not agree with you =)

Reply:
I guess that `.clang-format` just requires some modification to using with `clang-format 11`